### PR TITLE
Add possibility to change a note's collection

### LINKED ISSUE #112: Possibility to move a note to another notebook
This can be useful when reorganizing notes, or if we created it at the wrong place.

### PR DESCRIPTION
**Pull request by zecakeh**
… aka move notes.

In practice: copy the item into another collection and delete the old one.

This fixes #112.

Tested on web Firefox Linux
Tested on native Android

### CODE REVIEW

File: src/screens/NoteListScreen.tsx [hunk 1]
```diff
@@ -178,6 +178,7 @@ function RightAction(props: RightActionPropsType) {
   const viewSettings = useSelector((state: StoreState) => state.settings.viewSettings);
   const navigation = useNavigation<DefaultNavigationProp>();
   const { colUid } = props;
+  console.log("colUid", colUid);
```
Comment (tasn) [inline src/screens/NoteListScreen.tsx:181]: :thinking:

**Review by tasn — CHANGES_REQUESTED** [review]
I'd rather have it as an explicit move operation rather than just being able to change the collection and have this done, as it has non-obvious implications for now (the fact that it's deleted and a new one is created).

File: src/screens/NoteListScreen.tsx (hunk 1 above)
Comment (zecakeh) [reply src/screens/NoteListScreen.tsx:181]: Sorry 😅

**Comment by zecakeh** [comment]
I created a new screen to move the notes, I don't know if that's what you had in mind.

**Comment by tasn** [comment]
Yeah, that's what I had in mind! :)

**Pull request merged**